src/MergeContext.js

```diff
@@ -561,11 +566,36 @@ class CommitMessage
         return {name: this._customAuthor.name, email: this._customAuthor.email, date: this._defaultAuthor.date};
     }
 
-    // returns the position of the first non-ASCII_printable character (or -1)
-    _invalidCharacterPosition(str) {
+    // checks that the line represents only ASCII_printable characters
+    _parseRawCharacters(line) {
```

Comment:
Unlike other parsing methods, this one does not store the results:

```suggestion
    _checkRawCharacters(line) {
```

Reply:
Done (04f70b3).